kivakit-filesystems/java/src/test/java/com/telenav/kivakit/filesystem/java/JavaFolderTest.java

```diff
@@ -0,0 +1,60 @@
+package com.telenav.kivakit.filesystem.java;
+
+import com.telenav.kivakit.filesystem.spi.FileService;
+import com.telenav.kivakit.filesystem.spi.FolderService;
+import com.telenav.kivakit.kernel.interfaces.comparison.Matcher;
+import com.telenav.kivakit.resource.path.FileName;
+import com.telenav.kivakit.test.UnitTest;
+import org.junit.Test;
+
+import java.util.List;
+
+public class JavaFolderTest extends UnitTest {
+
+    public class JavaFileMatcher implements Matcher {
+
+        @Override
+        public boolean matches(Object o) {
+            return true;
+        }
+    }
+
+    @Test
+    public void testJavaFolder() {
+
+        try {
+            FolderService folder = new JavaFolder("/Users/yyzhou/jfs_test");
+            final boolean w = folder.isWritable();
+            System.out.println(w);
+
+//            FolderService folder1 = new JavaFolder("/Users/yyzhou/jfs_test1");
+//            folder.copyTo()
+
+            for (FileService f : folder.files()) {
```

Comment:
Change printlns to some kind of ensure()* call

Reply:
fixed